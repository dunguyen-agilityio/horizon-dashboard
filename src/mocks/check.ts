import { TCheck } from '@/types/check';

export const MOCK_CHECKS: TCheck[] = [
  {
    id: '1',
    name: 'Tony Reichert',
    progress: 45,
    quantity: 20,
    createdAt: new Date('2024-08-14T08:25:04.188Z'),
  },
  {
    id: '2',
    name: 'Zoey Lang',
    progress: 15,
    quantity: 20,
    createdAt: new Date('2024-08-14T08:25:04.188Z'),
  },
  {
    id: '3',
    name: 'Jane Fisher',
    progress: 55,
    quantity: 28,
    createdAt: new Date('2024-08-14T08:25:04.188Z'),
  },
  {
    id: '4',
    name: 'William Howard',
    progress: 65,
    quantity: 30,
    createdAt: new Date('2024-08-14T08:25:04.188Z'),
  },
];
