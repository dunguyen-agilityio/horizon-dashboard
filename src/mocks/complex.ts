import { Complex, ComplexStatus } from '@/types/complex';

export const MOCK_COMPLEXES: Complex[] = [
  {
    id: '1',
    name: 'Tony Reichert',
    progress: 45,
    status: ComplexStatus.APPROVED,
    createdAt: new Date('2024-08-14T08:25:04.188Z'),
  },
  {
    id: '2',
    name: 'Zoey Lang',
    progress: 15,
    status: ComplexStatus.DISABLE,
    createdAt: new Date('2024-08-14T08:25:04.188Z'),
  },
  {
    id: '3',
    name: 'Jane Fisher',
    progress: 55,
    status: ComplexStatus.ERROR,
    createdAt: new Date('2024-08-14T08:25:04.188Z'),
  },
  {
    id: '4',
    name: 'William Howard',
    progress: 65,
    status: ComplexStatus.APPROVED,
    createdAt: new Date('2024-08-14T08:25:04.188Z'),
  },
];
