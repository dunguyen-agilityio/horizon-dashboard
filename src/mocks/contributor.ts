import { Contributor } from '@/models/Contributor';
import { MOCK_USERS } from './user';

export const MOCK_CONTRIBUTORS = [
  {
    ...MOCK_USERS[0],
    rating: 23,
    template: 'Abstract Colors',
    createdAt: new Date('2024-01-13T07:25:04.188Z'),
  },
  {
    ...MOCK_USERS[1],
    rating: 80,
    template: 'Swipe Circles',
    createdAt: new Date('2024-08-13T07:25:04.188Z'),
  },
  {
    ...MOCK_USERS[2],
    template: 'Colorful Heaven',
    createdAt: new Date('2024-08-11T07:25:04.188Z'),
  },
  {
    ...MOCK_USERS[3],
    template: '3D Cubes Art',
    avatar: undefined,
  },
] as Contributor[];
