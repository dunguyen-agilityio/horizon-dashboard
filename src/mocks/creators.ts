export const MOCK_CREATORS = [
  {
    id: '1',
    avatar: '/default-avatar.webp',
    userName: 'maddison_c21',
    artwork: 9821,
    rating: 90,
  },
  {
    id: '2',
    avatar: '/default-avatar.webp',
    userName: 'karl.willO2',
    artwork: 7032,
    rating: 80,
  },
  {
    id: '3',
    avatar: '/default-avatar.webp',
    userName: 'andreea.1z',
    artwork: 5204,
    rating: 60,
  },
];
