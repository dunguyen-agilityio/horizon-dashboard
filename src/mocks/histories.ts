import { NFT_IMAGES } from '@/constants/images';

export const histories = [
  {
    image: NFT_IMAGES.DEFAULT,
    title: 'Colorful Heaven',
    author: 'Mark Benjamin',
    price: '1.33',
    timeAgo: new Date(Date.now() - 2 * 60 * 60 * 1000),
  },
  {
    image: NFT_IMAGES.DEFAULT,
    title: 'Abstract Colors',
    author: 'Esthera Jackson',
    price: '0.91',
    timeAgo: new Date(Date.now() - 6 * 60 * 60 * 1000),
  },
  {
    image: NFT_IMAGES.DEFAULT,
    title: 'ETH AI Brain',
    author: 'Nick Wilson',
    price: '2.82',
    timeAgo: new Date(Date.now() - 5 * 60 * 60 * 1000),
  },
  {
    image: NFT_IMAGES.DEFAULT,
    title: 'Swipe Circles',
    author: 'Peter Will',
    price: '2.32',
    timeAgo: new Date(Date.now() - 1.5 * 24 * 60 * 60 * 1000),
  },
];
