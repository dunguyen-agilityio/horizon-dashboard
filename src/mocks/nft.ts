// Types
import { NFTResponse, type NFTData } from '@/models/NFT';

// Constants
import { NFT_IMAGES } from '@/constants/images';

// Mocks
import { MOCK_IMAGE_RESPONSE, MOCK_USERS, MOCK_USERS_RESPONSE } from './user';
import { StrapiResponse } from '@/types/strapi';

const [MOCK_USER] = MOCK_USERS;

export const MOCK_NFTS: NFTData[] = [
  {
    id: '1',
    name: 'Mesh Gradients',
    author: MOCK_USER,
    price: 1.23,
    image: { url: NFT_IMAGES.DEFAULT, hash: 'mock hash' },
    members: MOCK_USERS,
    createdAt: new Date('2024-08-17T07:25:04.188Z'),
  },
];

export const MOCK_NFTS_RESPONSE: StrapiResponse<NFTResponse>[] = [
  {
    id: '1',
    attributes: {
      name: 'Mesh Gradients',
      author: { data: MOCK_USERS_RESPONSE[0] },
      price: 1.23,
      members: { data: MOCK_USERS_RESPONSE },
      image: { data: MOCK_IMAGE_RESPONSE },
      createdAt: new Date('2024-08-17T07:25:04.188Z'),
    },
  },
];
