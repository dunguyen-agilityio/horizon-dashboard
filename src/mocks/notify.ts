import { Notify } from '@/models/Notify';

// Type
import { INotification } from '@/types/notification';

export const MOCK_NOTIFIES: Notify[] = [
  new Notify({
    id: '1',
    message: 'Your task is scheduled to start in 10 minutes.',
    createdAt: new Date('2024-08-14T08:25:04.188Z'),
    link: '/kanban/1',
    unread: true,
  }),
  new Notify({
    id: '2',
    message: 'Your task is scheduled to start in 10 minutes.',
    createdAt: new Date('2024-08-13T07:25:04.188Z'),
    link: '/kanban/2',
    unread: true,
  }),
  new Notify({
    id: '3',
    message: 'Your task is scheduled to start in 10 minutes.',
    createdAt: new Date('2024-08-14T02:25:07.188Z'),
    link: '/kanban/3',
  }),
  new Notify({
    id: '4',
    message: 'Your task is scheduled to start in 10 minutes.',
    createdAt: new Date('2024-08-08T08:25:04.188Z'),
    link: '/kanban/4',
  }),
  new Notify({
    id: '5',
    message: 'Your task is scheduled to start in 10 minutes.',
    createdAt: new Date('2024-08-08T08:25:04.188Z'),
    link: '/kanban/5',
  }),
  new Notify({
    id: '6',
    message: 'Your task is scheduled to start in 10 minutes.',
    createdAt: new Date('2024-08-08T08:25:04.188Z'),
    link: '/kanban/6',
  }),
  new Notify({
    id: '7',
    message: 'Your task is scheduled to start in 10 minutes.',
    createdAt: new Date('2024-08-08T08:25:04.188Z'),
    link: '/kanban/7',
  }),
  new Notify({
    id: '8',
    message: 'Your task is scheduled to start in 10 minutes.',
    createdAt: new Date('2024-08-08T08:25:04.188Z'),
    link: '/kanban/8',
  }),
];

export const CONTROL_NOTIFICATIONS: INotification[] = [
  {
    id: '1',
    isSelected: true,
    isDisabled: false,
    label: 'Item update notifications',
  },
  {
    id: '2',
    isSelected: false,
    isDisabled: true,
    label: 'Item comment notifications',
  },
  {
    id: '3',
    isSelected: false,
    isDisabled: true,
    label: 'Buyer review notifications',
  },
  {
    id: '4',
    isSelected: false,
    isDisabled: true,
    label: 'Rating reminders notifications',
  },
  {
    id: '5',
    isSelected: false,
    isDisabled: true,
    label: 'Meetups near you notifications',
  },
  {
    id: '6',
    isSelected: false,
    isDisabled: true,
    label: 'Company news notifications',
  },
  {
    id: '7',
    isSelected: false,
    isDisabled: true,
    label: 'New launches and projects',
  },
  {
    id: '8',
    isSelected: false,
    isDisabled: true,
    label: 'Monthly product changes',
  },
  {
    id: '9',
    isSelected: false,
    isDisabled: true,
    label: 'Subscribe to newsletter',
  },
  {
    id: '10',
    isSelected: true,
    isDisabled: true,
    label: 'Email me when someone follows me',
  },
];
