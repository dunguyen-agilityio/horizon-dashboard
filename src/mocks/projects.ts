import { IProjectCardProps } from '@/ui/profile/PersonalProject/ProjectCard';

export const PROJECTS_MOCK: IProjectCardProps[] = [
  {
    baseProject: 'project #3',
    id: '2',
    title: 'Technology behind the Blockchain',
  },
  {
    baseProject: 'project #3',
    id: '1',
    title: 'Greatest way to a good Economy',
  },
  {
    baseProject: 'project #1',
    id: '3',
    title: 'Most essential tips for Burnout',
  },
];
