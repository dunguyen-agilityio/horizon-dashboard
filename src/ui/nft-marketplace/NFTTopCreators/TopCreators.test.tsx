import { render, screen } from '@testing-library/react';
import TopCreators from '.';
import { MOCK_CREATORS } from '@/mocks/creators';

describe('TopCreators tests', () => {
  const setup = () => render(<TopCreators data={MOCK_CREATORS} />);

  it('Should match snapshot', () => {
    const { container } = setup();
    expect(container).toMatchSnapshot();
  });

  it('renders empty content when there is no data', () => {
    render(<TopCreators data={[]} />);

    expect(screen.getByText('No creator to display.')).toBeInTheDocument();
  });
});
