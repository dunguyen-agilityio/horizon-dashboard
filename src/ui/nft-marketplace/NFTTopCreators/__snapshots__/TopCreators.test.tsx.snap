// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopCreators tests Should match snapshot 1`] = `
<div>
  <div
    class="flex flex-col bg-white dark:bg-indigo-light rounded-md w-full 4xl:max-w-nftCard overflow-auto shadow-md pt-[19px]"
  >
    <div
      class="flex items-center justify-between p-[18px]"
    >
      <p
        class="text-primary dark:text-white text-medium leading-8 font-poppins font-bold text-blue-450"
      >
        Top Creators
      </p>
      <button
        class="z-0 group relative inline-flex items-center justify-center box-border appearance-none select-none whitespace-nowrap subpixel-antialiased overflow-hidden tap-highlight-transparent data-[pressed=true]:scale-[0.97] outline-none data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2 px-4 min-w-20 h-10 text-small gap-2 [&>svg]:max-w-[theme(spacing.8)] transition-transform-colors-opacity motion-reduce:transition-none data-[hover=true]:opacity-hover text-blue-450 dark:text-white font-medium bg-gray dark:bg-purple-750 rounded-[70px]"
        type="button"
      >
        See all
      </button>
    </div>
    <div
      class="flex flex-col relative gap-4 w-full mt-3 first:[&_*:is(th,td)]:pr-0 [&_*:is(th,td)]:bg-transparent [&_*:is(th,td)]:dark:transparent [&>div]:p-0 [&>div]:bg-transparent [&>div]:shadow-none [&_*:is(th,td)]:pt-2 [&_*:is(th,td)]:pb-4 [&>div]:overflow-visible"
    >
      <div
        class="p-4 z-0 flex flex-col relative justify-between gap-4 bg-content1 overflow-auto rounded-large shadow-small w-full"
      >
        <table
          aria-describedby=""
          class="min-w-full h-auto table-auto w-full"
          id="react-aria-:r0:"
          role="grid"
          tabindex="0"
        >
          <thead
            class="[&>tr]:first:rounded-lg bg-transparent"
            role="rowgroup"
          >
            <tr
              class="group outline-none data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2"
              role="row"
            >
              <th
                class="group px-3 h-10 align-middle bg-default-100 whitespace-nowrap text-foreground-500 text-tiny font-semibold first:rounded-l-lg rtl:first:rounded-r-lg rtl:first:rounded-l-[unset] last:rounded-r-lg rtl:last:rounded-l-lg rtl:last:rounded-r-[unset] data-[sortable=true]:cursor-pointer data-[hover=true]:text-foreground-400 outline-none data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2 text-start pl-7"
                data-key="userName"
                id="react-aria-:r0:-userName"
                role="columnheader"
                tabindex="-1"
              >
                <p
                  class="text-secondary dark:text-white text-small font-medium"
                >
                  Name
                </p>
              </th>
              <th
                class="group px-3 h-10 align-middle bg-default-100 whitespace-nowrap text-foreground-500 text-tiny font-semibold first:rounded-l-lg rtl:first:rounded-r-lg rtl:first:rounded-l-[unset] last:rounded-r-lg rtl:last:rounded-l-lg rtl:last:rounded-r-[unset] data-[sortable=true]:cursor-pointer data-[hover=true]:text-foreground-400 outline-none data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2 text-start"
                data-key="artwork"
                id="react-aria-:r0:-artwork"
                role="columnheader"
                tabindex="-1"
              >
                <p
                  class="text-secondary dark:text-white text-small font-medium"
                >
                  Artworks
                </p>
              </th>
              <th
                class="group px-3 h-10 align-middle bg-default-100 whitespace-nowrap text-foreground-500 text-tiny font-semibold first:rounded-l-lg rtl:first:rounded-r-lg rtl:first:rounded-l-[unset] last:rounded-r-lg rtl:last:rounded-l-lg rtl:last:rounded-r-[unset] data-[sortable=true]:cursor-pointer data-[hover=true]:text-foreground-400 outline-none data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2 text-start"
                data-key="rating"
                id="react-aria-:r0:-rating"
                role="columnheader"
                tabindex="-1"
              >
                <p
                  class="text-secondary dark:text-white text-small font-medium"
                >
                  Rating
                </p>
              </th>
            </tr>
            <tr
              aria-hidden="true"
              class="w-px h-px block"
              style="margin-left: 0.25rem; margin-top: 0.25rem;"
              tabindex="-1"
            />
          </thead>
          <tbody
            class="bg-gray dark:bg-indigo"
            role="rowgroup"
          >
            <tr
              aria-labelledby="react-aria-:r0:-1-userName"
              class="group outline-none data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2"
              data-first="true"
              data-key="1"
              role="row"
              tabindex="-1"
            >
              <td
                class="py-2 relative align-middle whitespace-normal text-small font-normal [&>*]:z-1 [&>*]:relative outline-none data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2 before:content-[''] before:absolute before:z-0 before:inset-0 before:opacity-0 data-[selected=true]:before:opacity-100 group-data-[disabled=true]:text-foreground-300 group-data-[disabled=true]:cursor-not-allowed before:bg-default/60 data-[selected=true]:text-default-foreground first:before:rounded-l-lg rtl:first:before:rounded-r-lg rtl:first:before:rounded-l-[unset] last:before:rounded-r-lg rtl:last:before:rounded-l-lg rtl:last:before:rounded-r-[unset] text-start px-0 group-aria-[selected=false]:group-data-[hover=true]:bg-transparent"
                data-key="11userName"
                id="react-aria-:r0:-1-userName"
                role="rowheader"
                tabindex="-1"
              >
                <div
                  class="flex items-center pl-7"
                >
                  <span
                    class="flex relative justify-center items-center box-border overflow-hidden align-middle z-0 outline-none data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2 w-8 h-8 text-tiny bg-default text-default-foreground rounded-full mr-2"
                    tabindex="-1"
                  >
                    <img
                      alt="maddison_c21"
                      class="flex object-cover w-full h-full transition-opacity !duration-500 opacity-0 data-[loaded=true]:opacity-100"
                      src="https://res.cloudinary.com/dn5vw6fjp/image/upload/v1726630608/image_de1e29b0b7.png"
                    />
                  </span>
                  <span
                    class="text-primary dark:text-white text-small font-medium bg-transparent text-primary"
                  >
                    @maddison_c21
                  </span>
                </div>
              </td>
              <td
                class="py-2 px-3 relative align-middle whitespace-normal text-small font-normal [&>*]:z-1 [&>*]:relative outline-none data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2 before:content-[''] before:absolute before:z-0 before:inset-0 before:opacity-0 data-[selected=true]:before:opacity-100 group-data-[disabled=true]:text-foreground-300 group-data-[disabled=true]:cursor-not-allowed before:bg-default/60 data-[selected=true]:text-default-foreground first:before:rounded-l-lg rtl:first:before:rounded-r-lg rtl:first:before:rounded-l-[unset] last:before:rounded-r-lg rtl:last:before:rounded-l-lg rtl:last:before:rounded-r-[unset] text-start group-aria-[selected=false]:group-data-[hover=true]:bg-transparent"
                data-key="11artwork"
                role="gridcell"
                tabindex="-1"
              >
                <span
                  class="text-primary dark:text-white text-small font-medium bg-transparent text-secondary"
                >
                  9,821
                </span>
              </td>
              <td
                class="py-2 px-3 relative align-middle whitespace-normal text-small font-normal [&>*]:z-1 [&>*]:relative outline-none data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2 before:content-[''] before:absolute before:z-0 before:inset-0 before:opacity-0 data-[selected=true]:before:opacity-100 group-data-[disabled=true]:text-foreground-300 group-data-[disabled=true]:cursor-not-allowed before:bg-default/60 data-[selected=true]:text-default-foreground first:before:rounded-l-lg rtl:first:before:rounded-r-lg rtl:first:before:rounded-l-[unset] last:before:rounded-r-lg rtl:last:before:rounded-l-lg rtl:last:before:rounded-r-[unset] text-start group-aria-[selected=false]:group-data-[hover=true]:bg-transparent"
                data-key="11rating"
                role="gridcell"
                tabindex="-1"
              >
                <div
                  aria-label="Rating"
                  aria-valuemax="100"
                  aria-valuemin="0"
                  aria-valuenow="90"
                  aria-valuetext="90%"
                  class="flex flex-col gap-2 w-full pr-3"
                  id="react-aria-:r2:"
                  role="progressbar"
                >
                  <div
                    class="flex justify-between"
                  >
                    <span
                      class="text-small"
                    >
                      90%
                    </span>
                  </div>
                  <div
                    class="z-0 relative bg-default-300/50 overflow-hidden h-1 rounded-full"
                  >
                    <div
                      class="h-full rounded-full transition-transform !duration-500 bg-blue-450 dark:bg-purple-750"
                      style="transform: translateX(-100%);"
                    />
                  </div>
                </div>
              </td>
            </tr>
            <tr
              aria-labelledby="react-aria-:r0:-2-userName"
              class="group outline-none data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2"
              data-key="2"
              data-middle="true"
              data-odd="true"
              role="row"
              tabindex="-1"
            >
              <td
                class="py-2 relative align-middle whitespace-normal text-small font-normal [&>*]:z-1 [&>*]:relative outline-none data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2 before:content-[''] before:absolute before:z-0 before:inset-0 before:opacity-0 data-[selected=true]:before:opacity-100 group-data-[disabled=true]:text-foreground-300 group-data-[disabled=true]:cursor-not-allowed before:bg-default/60 data-[selected=true]:text-default-foreground first:before:rounded-l-lg rtl:first:before:rounded-r-lg rtl:first:before:rounded-l-[unset] last:before:rounded-r-lg rtl:last:before:rounded-l-lg rtl:last:before:rounded-r-[unset] text-start px-0 group-aria-[selected=false]:group-data-[hover=true]:bg-transparent"
                data-key="22userName"
                id="react-aria-:r0:-2-userName"
                role="rowheader"
                tabindex="-1"
              >
                <div
                  class="flex items-center pl-7"
                >
                  <span
                    class="flex relative justify-center items-center box-border overflow-hidden align-middle z-0 outline-none data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2 w-8 h-8 text-tiny bg-default text-default-foreground rounded-full mr-2"
                    tabindex="-1"
                  >
                    <img
                      alt="karl.willO2"
                      class="flex object-cover w-full h-full transition-opacity !duration-500 opacity-0 data-[loaded=true]:opacity-100"
                      src="https://res.cloudinary.com/dn5vw6fjp/image/upload/v1726630608/image_de1e29b0b7.png"
                    />
                  </span>
                  <span
                    class="text-primary dark:text-white text-small font-medium bg-transparent text-primary"
                  >
                    @karl.willO2
                  </span>
                </div>
              </td>
              <td
                class="py-2 px-3 relative align-middle whitespace-normal text-small font-normal [&>*]:z-1 [&>*]:relative outline-none data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2 before:content-[''] before:absolute before:z-0 before:inset-0 before:opacity-0 data-[selected=true]:before:opacity-100 group-data-[disabled=true]:text-foreground-300 group-data-[disabled=true]:cursor-not-allowed before:bg-default/60 data-[selected=true]:text-default-foreground first:before:rounded-l-lg rtl:first:before:rounded-r-lg rtl:first:before:rounded-l-[unset] last:before:rounded-r-lg rtl:last:before:rounded-l-lg rtl:last:before:rounded-r-[unset] text-start group-aria-[selected=false]:group-data-[hover=true]:bg-transparent"
                data-key="22artwork"
                role="gridcell"
                tabindex="-1"
              >
                <span
                  class="text-primary dark:text-white text-small font-medium bg-transparent text-secondary"
                >
                  7,032
                </span>
              </td>
              <td
                class="py-2 px-3 relative align-middle whitespace-normal text-small font-normal [&>*]:z-1 [&>*]:relative outline-none data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2 before:content-[''] before:absolute before:z-0 before:inset-0 before:opacity-0 data-[selected=true]:before:opacity-100 group-data-[disabled=true]:text-foreground-300 group-data-[disabled=true]:cursor-not-allowed before:bg-default/60 data-[selected=true]:text-default-foreground first:before:rounded-l-lg rtl:first:before:rounded-r-lg rtl:first:before:rounded-l-[unset] last:before:rounded-r-lg rtl:last:before:rounded-l-lg rtl:last:before:rounded-r-[unset] text-start group-aria-[selected=false]:group-data-[hover=true]:bg-transparent"
                data-key="22rating"
                role="gridcell"
                tabindex="-1"
              >
                <div
                  aria-label="Rating"
                  aria-valuemax="100"
                  aria-valuemin="0"
                  aria-valuenow="80"
                  aria-valuetext="80%"
                  class="flex flex-col gap-2 w-full pr-3"
                  id="react-aria-:r5:"
                  role="progressbar"
                >
                  <div
                    class="flex justify-between"
                  >
                    <span
                      class="text-small"
                    >
                      80%
                    </span>
                  </div>
                  <div
                    class="z-0 relative bg-default-300/50 overflow-hidden h-1 rounded-full"
                  >
                    <div
                      class="h-full rounded-full transition-transform !duration-500 bg-blue-450 dark:bg-purple-750"
                      style="transform: translateX(-100%);"
                    />
                  </div>
                </div>
              </td>
            </tr>
            <tr
              aria-labelledby="react-aria-:r0:-3-userName"
              class="group outline-none data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2"
              data-key="3"
              data-last="true"
              role="row"
              tabindex="-1"
            >
              <td
                class="py-2 relative align-middle whitespace-normal text-small font-normal [&>*]:z-1 [&>*]:relative outline-none data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2 before:content-[''] before:absolute before:z-0 before:inset-0 before:opacity-0 data-[selected=true]:before:opacity-100 group-data-[disabled=true]:text-foreground-300 group-data-[disabled=true]:cursor-not-allowed before:bg-default/60 data-[selected=true]:text-default-foreground first:before:rounded-l-lg rtl:first:before:rounded-r-lg rtl:first:before:rounded-l-[unset] last:before:rounded-r-lg rtl:last:before:rounded-l-lg rtl:last:before:rounded-r-[unset] text-start px-0 group-aria-[selected=false]:group-data-[hover=true]:bg-transparent"
                data-key="33userName"
                id="react-aria-:r0:-3-userName"
                role="rowheader"
                tabindex="-1"
              >
                <div
                  class="flex items-center pl-7"
                >
                  <span
                    class="flex relative justify-center items-center box-border overflow-hidden align-middle z-0 outline-none data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2 w-8 h-8 text-tiny bg-default text-default-foreground rounded-full mr-2"
                    tabindex="-1"
                  >
                    <img
                      alt="andreea.1z"
                      class="flex object-cover w-full h-full transition-opacity !duration-500 opacity-0 data-[loaded=true]:opacity-100"
                      src="https://res.cloudinary.com/dn5vw6fjp/image/upload/v1726630608/image_de1e29b0b7.png"
                    />
                  </span>
                  <span
                    class="text-primary dark:text-white text-small font-medium bg-transparent text-primary"
                  >
                    @andreea.1z
                  </span>
                </div>
              </td>
              <td
                class="py-2 px-3 relative align-middle whitespace-normal text-small font-normal [&>*]:z-1 [&>*]:relative outline-none data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2 before:content-[''] before:absolute before:z-0 before:inset-0 before:opacity-0 data-[selected=true]:before:opacity-100 group-data-[disabled=true]:text-foreground-300 group-data-[disabled=true]:cursor-not-allowed before:bg-default/60 data-[selected=true]:text-default-foreground first:before:rounded-l-lg rtl:first:before:rounded-r-lg rtl:first:before:rounded-l-[unset] last:before:rounded-r-lg rtl:last:before:rounded-l-lg rtl:last:before:rounded-r-[unset] text-start group-aria-[selected=false]:group-data-[hover=true]:bg-transparent"
                data-key="33artwork"
                role="gridcell"
                tabindex="-1"
              >
                <span
                  class="text-primary dark:text-white text-small font-medium bg-transparent text-secondary"
                >
                  5,204
                </span>
              </td>
              <td
                class="py-2 px-3 relative align-middle whitespace-normal text-small font-normal [&>*]:z-1 [&>*]:relative outline-none data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2 before:content-[''] before:absolute before:z-0 before:inset-0 before:opacity-0 data-[selected=true]:before:opacity-100 group-data-[disabled=true]:text-foreground-300 group-data-[disabled=true]:cursor-not-allowed before:bg-default/60 data-[selected=true]:text-default-foreground first:before:rounded-l-lg rtl:first:before:rounded-r-lg rtl:first:before:rounded-l-[unset] last:before:rounded-r-lg rtl:last:before:rounded-l-lg rtl:last:before:rounded-r-[unset] text-start group-aria-[selected=false]:group-data-[hover=true]:bg-transparent"
                data-key="33rating"
                role="gridcell"
                tabindex="-1"
              >
                <div
                  aria-label="Rating"
                  aria-valuemax="100"
                  aria-valuemin="0"
                  aria-valuenow="60"
                  aria-valuetext="60%"
                  class="flex flex-col gap-2 w-full pr-3"
                  id="react-aria-:r8:"
                  role="progressbar"
                >
                  <div
                    class="flex justify-between"
                  >
                    <span
                      class="text-small"
                    >
                      60%
                    </span>
                  </div>
                  <div
                    class="z-0 relative bg-default-300/50 overflow-hidden h-1 rounded-full"
                  >
                    <div
                      class="h-full rounded-full transition-transform !duration-500 bg-blue-450 dark:bg-purple-750"
                      style="transform: translateX(-100%);"
                    />
                  </div>
                </div>
              </td>
            </tr>
          </tbody>
        </table>
      </div>
    </div>
  </div>
</div>
`;
